can_explorer_client/.eslintcache

```diff
@@ -0,0 +1 @@
+[{"D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\index.js":"1","D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\reportWebVitals.js":"2","D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\App.js":"3"},{"size":500,"mtime":499162500000,"results":"4","hashOfConfig":"5"},{"size":362,"mtime":499162500000,"results":"6","hashOfConfig":"5"},{"size":140,"mtime":1611431956708,"results":"7","hashOfConfig":"5"},{"filePath":"8","messages":"9","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"e18iux",{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\index.js",[],"D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\reportWebVitals.js",[],"D:\\Waterloo Computer Engineering\\2AMidnightsun\\CAN-Explorer\\can_explorer_client\\src\\App.js",[]]
```

Comment:
Looks like there are some unneeded files from the cache here, could these be removed.